herddb-docker/pom.xml

```diff
@@ -149,8 +108,11 @@
               </to>
               <container>
                 <creationTime>USE_CURRENT_TIMESTAMP</creationTime>
-                <extraClasspath>${image.workdir}/custom/*</extraClasspath>
-                <mainClass>herddb.server.ServerMain</mainClass>
+                <entrypoint>/opt/herddb/bin/service</entrypoint>
```

Comment:
generally speaking it is always better to bypass scripts  since it ensures the process #1 is `java` and there is no issue in any container environment. With recent jib version, jib will ensure the classpath is in a file so it is set as `-cp @jib-classpath-file` so easy to override in k8s or any orchestrator when needed. Using a script always indirects it and can create issues if not well tested in the build (`exec`/`nohup` evolutions in particular) so is more risky IMHO.